pyth_observer/calendar.py

```diff
@@ -30,10 +30,10 @@
 
 
 class HolidayCalendar:
-    def is_market_open(self, asset_type, dt):
-        # equity market
+    def is_market_open(self, asset_type: str, dt: datetime.datetime):
```

Comment:
For the record, the quantlib library (which has Python bindings) has a robust implementation of "is the market open?" for markets across many countries (e.g. [see this](https://quantlib.js.org/docs/classes/_ql_time_calendar_.calendar.html)).